pkg/controllers/vdb/podfacts.go

```diff
@@ -601,6 +608,16 @@ func (p *PodFacts) checkIsDBCreated(ctx context.Context, vdb *vapi.VerticaDB, pf
 	return nil
 }
 
+// checkIfAgentRunning will check if the Vertica agent is running and set state in pf.agentRunning
+func (p *PodFacts) checkIfAgentRunning(ctx context.Context, vdb *vapi.VerticaDB, pf *PodFact, gs *GatherState) error {
```

Comment:
I don't think we need this function. There is a mapping function from the `GatherState` to `PodFact` (see `checkForSimpleGatherStateMapping`). We can just update that.